### ISSUE TITLE
Implement @ character for Context variable binding

### ISSUE THREAD

**Issue by wnm3** [IBM/JSONata4Java#48]
See https://docs.jsonata.org/path-operators#-context-variable-binding

**Comment by aubelsb2** [comment]
Is this still being considered?

**Comment by wnm3** [comment]
Yes, I hope to address all these open issues. There is a bigger issue though in the example shown on the jsonata documentation URL above -- at present, the parser isn't evaluating the object declaration using { and } so we would still not be able to perform what is show there.
